src/workers/jobs.js

```diff
@@ -14,6 +14,18 @@ import { Notifications, sendUserNotification } from '../server/notifications'
 
 const zipMemoization = {}
 
+function optOutsByOrgId(orgId) {
+  return r.knex.select('cell').from('opt_out').where('organization_id', orgId)
+}
+
+function optOutsByInstance() {
+  return r.knex.select('cell').from('opt_out')
+}
+
+function getOptOutCount(orgId) {
```

Comment:
can we rename this something like `getOptOutSubQuery` since it's not the count, but the full list.